src/js.rs

```diff
@@ -682,6 +682,15 @@ extern {
     pub fn delete(this: &WeakSet, value: Object) -> bool;
 }
 
+// JSON
+#[wasm_bindgen]
+extern {
+    pub type JSON;
```

Comment:
Rust style for acronyms within identifiers would be to use `Json` here:

```rust
#[js_name = JSON]
pub type Json;
```

The other option, since `JSON` is sort of a module/namespace in JS, would be to add bindings like this:

```rust
mod json {
    #[wasm_bindgen]
    extern {
        #[wasm_bindgen(catch, js_namespace = JSON)]
        pub fn stringify(value: JsValue) -> Result<JsString, JsValue>;
    }
}
```